test/Integration/plugin-test.js

```diff
@@ -473,6 +473,18 @@ lab.experiment('plugin', () => {
     expect(response.statusCode).to.equal(200);
   });
 
+  lab.test('displayRequestDuration true', async () => {
+    const server = await Helper.createServer({ displayRequestDuration: true });
```

Comment:
@CuteTechGuy I suppose that provided tests are not enough. First of all, need to provide testing of the resulted JSON with the right data and structure.